app/cli.php

```diff
@@ -1,6 +1,6 @@
 <?php
-
 require_once __DIR__.'/init.php';
+require_once __DIR__.'/controllers/general.php';
```

Comment:
This should be in `specs.php` since it adds like 2-4 seconds delay for every CLI command 👍🏻

Reply:
nvm, looks like my docker was just a bit funky and slow